.gitignore

```diff
@@ -8,3 +8,9 @@ build/
 *.dll
 *.dylib
 hs_err*.log
+
```

Comment:
I use IntelliJ rather than Eclipse, and there are a different set of local user-specific project files that should be ignored by Git